modules/Makefile.circleci-2.0

```diff
@@ -59,6 +59,11 @@ circle\:deploy-kubernetes:
 	@sleep 3
 	@$(SELF) kubernetes:list-deployments kubernetes:list-rs kubernetes:list-pods
 
+## Cleanup docker images on CircleCI
+circle\:cleanup:
```

Comment:
nit: `circle:cleanup-docker` would be clearer

Reply:
on it